### ISSUE TITLE
iSponsorBlockTV also blocks google tv and android tv

### ISSUE THREAD

**Issue by bruvv** [gabe565/CastSponsorSkip#46]
https://github.com/dmunozv04/iSponsorBlockTV
Can you perhaps include this in here :) ? or work with @dmunozv04 ? Would be awesome!

**Comment by gabe565** [comment]
@bruvv CastSponsorSkip actually supports Google TV and Android TV also! At least, it works for my NVIDIA Shield. I should add that to the readme. I can also add a link to iSponsorBlockTV to the bottom!

**Comment by dmunozv04** [comment]
Having more choices is always a good thing!

**Comment by gabe565** [comment]
Agreed. iSponsorBlockTV seems quite a bit heavier, but it supports a lot more device types, so it just depends on the user's needs.

**Comment by dmunozv04** [comment]
Yep, most of it is just python dependencies

**Comment by bruvv** [comment]
It's. But I was taking about the inbuilt YouTube stuff like webs

**Comment by diamant-x** [comment]
Hi, Coming from the related post.
I can confirm the native Youtube Android TV app is detected by the code. I had to 1) generate an API key for it AND 2) enable Youtube data api V3 (not required for my chromecast casted videos).

However, while in my chromecast i believe the addon would 'auto-skip' pre-roll ads (as like clicking on the 'skip now' button after the countdown expires) I just tested it and it did not skip the ad in the youtube app from my nvidia shield.
Not sure if that's something that is now the new normal?

PD: I have since installed the SmartTube app which skips everything, no UI-menu ads and can force always to maximum resolution.

**Comment by bruvv** [comment]
@diamant-x can you define it a bit more? Nvidia Shield is not the same as for example the webos YouTube or Samsung tizen version. Would that work too ?

**Comment by diamant-x** [comment]
> @diamant-x can you define it a bit more? Nvidia Shield is not the same as for example the webos YouTube or Samsung tizen version. Would that work too ?

My test was done in a Nvidia Shield (2017) using the official Youtube application, I believe it runs native Android TV, (not google TV). Not sure if that was the information missing?

**Comment by gabe565** [comment]
@diamant-x I've seen some other reports of ad-skip not working on newer devices. Unfortunately it seems like the necessary API for ad-skip isn't available on some devices, and I haven't been able to find an alternative so far.

SmartTube is a great app! I'm glad it's working for you :)
